src/lib/stores/Stats.ts

```diff
@@ -1,26 +1,33 @@
-import { derived, writable, type Readable, type Writable } from 'svelte/store';
+import { get, derived, writable, type Readable, type Writable } from 'svelte/store';
 
-import type { Stat } from '$lib/types';
 import { branding } from '$lib/configuration';
-import { getStatList, promiseList } from '$lib/utilities/StatBuilder';
-import { loadResources } from '$lib/stores/Resources';
+import { getStatList, StatPromise } from '$lib/utilities/StatBuilder';
+
+import type { StatResult } from '$lib/models/Stat';
+import { loading as resourcesPromise } from '$lib/stores/Resources';
 
 export const hasError: Writable<boolean> = writable(false);
 export const loaded: Writable<boolean> = writable(false);
-const statData: Writable<Stat[]> = writable([]);
-export const stats: Readable<Stat[]> = derived(statData, ($s) => $s.filter((stat) => !stat.auth));
-export const authStats: Readable<Stat[]> = derived(statData, ($s) =>
+const statData: Writable<StatResult[]> = writable([]);
+export const stats: Readable<StatResult[]> = derived(statData, ($s) =>
+  $s.filter((stat) => !stat.auth),
+);
+export const authStats: Readable<StatResult[]> = derived(statData, ($s) =>
   $s.filter((stat) => stat.auth),
 );
 
 export async function loadLandingStats() {
-  await loadResources();
+  if (get(statData).length > 0) {
+    return;
+  }
```

Comment:
Does this load the open and auth once a user in BDC is logged in? Is the redirect enough to cause this to pull the right data?

Might revisit - as a followup pr?